src/components/MagicCodeInput.js

```diff
@@ -258,7 +258,13 @@ function MagicCodeInput(props) {
                         {/* Hide the input above the text. Cannot set opacity to 0 as it would break pasting on iOS Safari. */}
                         <View style={[StyleSheet.absoluteFillObject, styles.w100, styles.bgTransparent]}>
                             <TextInput
-                                ref={(ref) => (inputRefs.current[index] = ref)}
+                                ref={(ref) => {
+                                    inputRefs.current[index] = ref;
+                                    // Setting Attribute type search to prvent Password Manager to appear in Mobile Chrome
```

Comment:
```suggestion
                                    // Setting attribute type to "search" to prevent Password Manager from appearing in Mobile Chrome
```